base/changes.txt

```diff
@@ -58,6 +60,9 @@ not part of the distribution.
 	* usrguide.tex (subsection{Performance}):
 	Document efficiency of ltcmd definitions
 
+2023-11-26  David Carlisle  <David.Carlisle@latex-project.org>
```

Comment:
I would change the date to today (and also elsewhere) because it is kind of weird if an update shows up much earlier in the list compared to it actually showing up in the -dev format

Reply:
remark: in other cases (like the pr I did yester which was also stale) there have been intermediate changes so date and/or version number were already updated and in that cases it becomes really confusing to keep the original text.